pkg/web_css/lib/src/_form.scss

```diff
@@ -80,6 +80,6 @@
   }
 }
 
-#-admin-set-publisher-input {
+#-admin-set-publisher-input, #-admin-retract-package-version-input, #-admin-restore-retract-package-version-input{
```

Comment:
Break lines... or maybe make a class called `-admin-dropdown` since we are using this on multiple dropdowns now..

Reply:
done